src/resources/postcss/tickets-admin.pcss

```diff
@@ -617,3 +617,15 @@ p.description {
 #ticket_form .original_stock_cap span {
 	margin-right: 0;
 }
+
+.ticket-editor-notice {
+	padding: 5px;
+	margin-bottom: 10px;
+
+	&.info {
+		background-color: #EDF6FC;
```

Comment:
Is this covered by a common utilities var?

Reply:
Nope, I found none for this specific color.

Reply:
@paulmskim should we add it to common utilities or leave this here for now?

Reply:
This is a better question for design and how it fits into the larger design system, but considering this is the admin it's likely not something we'd add to the utilities.